resources/definitions/fdmprinter.def.json

```diff
@@ -699,6 +699,20 @@
                                 }
                             }
                         },
+                        "topmost_skin_line_width":
+                        {
+                            "label": "Roofing Line Width",
+                            "description": "Width of a single line of the areas at the top of the print.",
+                            "unit": "mm",
+                            "minimum_value": "0.001",
+                            "minimum_value_warning": "0.1 + 0.4 * machine_nozzle_size",
+                            "maximum_value_warning": "2 * machine_nozzle_size",
+                            "default_value": 0.4,
+                            "type": "float",
+                            "value": "line_width",
```

Comment:
I don't see topmost skin as a subpart of normal skin. It's a separate feature on its own. It should therefore inherit from the line width like any other print feature line width setting.